lib/sql.js

```diff
@@ -601,8 +623,33 @@ SQLConnector.prototype.updateAttributes = function(model, id, data, options, cb)
  * @param {Function} cb The callback function
  * @returns {ParameterizedSQL} The UPDATE SQL statement
  */
-SQLConnector.prototype.buildUpdate = function(model, where, data, options) {
+SQLConnector.prototype._buildUpdate = function(model, where, data, options) {
```

Comment:
Good point; that was Miroslav's concern too (please see [this comment](https://github.com/strongloop/loopback-connector/pull/35#issuecomment-170007325)) I thought it should start the name with `_` to be consistent with other private methods; Also Miroslav suggested  [here](https://github.com/strongloop/loopback-connector/pull/35#discussion-diff-49188374) we can have it with underscore, but also we should keep `buildUpdate` as an alias of` _buildUpdate`, with a deprecation notice. So should I revert it back to `buildUpdate` or you that's fine as long as I have an alias for it?